readthedocs/organizations/signals.py

```diff
@@ -74,3 +76,12 @@ def remove_organization_completely(sender, instance, using, **kwargs):
         version.delete()
 
     projects.delete()
+
+
+@receiver(build_complete, sender=Build)
+def mark_organization_assets_not_cleaned(sender, build, **kwargs):
+    """Mark the organization assets as not cleaned if there is a new build."""
+    organization = build.project.organizations.first()
+    if build.state == BUILD_STATE_FINISHED and build.success and organization:
+        organization.artifacts_cleaned = False
```

Comment:
Believe this is run on the builder, so we need to do this via API.

Reply:
Good point! This may not be trivial work then, since I don't think we can do this with APIv2 and we are using APIv3 from the builder yet. I'll think more about this before doing the work.

Reply:
I'd just trigger a task that runs on the web, as the simplest path.

Reply:
Sounds simple and smart to me 😄